heartbeat/config/config.go

```diff
@@ -27,13 +27,12 @@ import (
 
 // Config defines the structure of heartbeat.yml.
 type Config struct {
-	RunOnce         bool                 `config:"run_once"`
-	Monitors        []*conf.C            `config:"monitors"`
-	ConfigMonitors  *conf.C              `config:"config.monitors"`
-	Scheduler       Scheduler            `config:"scheduler"`
-	Autodiscover    *autodiscover.Config `config:"autodiscover"`
-	SyntheticSuites []*conf.C            `config:"synthetic_suites"`
```

Comment:
Removed this line, this is from some very early alphas where we had a separate config entry